### PR TITLE
ODBC: Document the new VarcharLimit and EarlyExecution params

### PR DESCRIPTION
**Pull request by bpintea**
Add the documentation for the newly added [VarcharLimit](https://github.com/elastic/elasticsearch-sql-odbc/pull/229) and
[EarlyExecution](https://github.com/elastic/elasticsearch-sql-odbc/pull/227) DSN attributes.

### CODE REVIEW

**Review by matriv — APPROVED** [review]

File: docs/reference/sql/endpoints/odbc/configuration.asciidoc [hunk 1]
```diff
@@ -199,6 +199,11 @@ This corresponds to the `MaxFetchSize` setting in <<odbc-cfg-dsnparams>>.
 The maximum number of megabytes that the driver will accept for one page.
 This corresponds to the `MaxBodySizeMB` setting in <<odbc-cfg-dsnparams>>.
 +
+* Varchar limit
++
+The maximum caracter length of the string type columns.
```
Comment (codebrain) [inline docs/reference/sql/endpoints/odbc/configuration.asciidoc:204]:
```suggestion
The maximum character length of the string type columns.
```

File: docs/reference/sql/endpoints/odbc/configuration.asciidoc [hunk 2]
```diff
@@ -402,6 +413,18 @@ This is concerning the HTTP answer body of one page, not the cumulated data
...
 
+`VarcharLimit` (default: `0`)::
+The maximum width of the string columns.
+If this setting is greater than zero, the driver will advertise all the string
+type columns as having a maximum character length equal to this value and will
+truncate any longer string to it. The string types are textual fields
+(TEXT, KEYWORD etc.) and some specialized fields (IP, the GEOs etc.). Note that
+no interpretation of the value is performed before trunctation, which can lead
+to invalid values if the limit is set too low.
+This is required for those applications that do not support column lenghts as
```
Comment (codebrain) [inline docs/reference/sql/endpoints/odbc/configuration.asciidoc:424]:
```suggestion
This is required for those applications that do not support column lengths as
```

**Review by codebrain — CHANGES_REQUESTED** [review]
LGTM, couple of typos

**Comment by bpintea** [comment]
Thanks, @codebrain , @matriv .

**Pull request merged**